### PR TITLE
Update --label definition for consistency

### PR DESCRIPTION
**Pull request by stevenGravy**
correct spelling for `diag-addr` description and made labels consistent with roles description. Also added an example on specifying labels as the `:` usage in YAML vs `=` in CLI caused some confusion.

### CODE REVIEW

File: docs/5.0/pages/cli-docs.mdx [hunk 1]
```diff
@@ -41,7 +41,7 @@ the following services.
 | `--nodename` | `hostname` command on the machine | **string** | assigns an alternative name for the node which can be used by clients to login. By default it's equal to the value returned by
 | `-c, --config` | `/etc/teleport.yaml` | **string** `.yaml` filepath | starts services with config specified in the YAML file, overrides CLI flags if set
 | `--bootstrap` | none | **string** `.yaml` filepath | bootstrap configured YAML resources {/* TODO link how to configure this file */}
-| `--labels` | none | **string** comma-separated list | assigns a set of labels to a node. See the explanation of labeling mechanism in the [Labeling Nodes](admin-guide.mdx#labeling-nodes-and-applications) section.
+| `--labels` | none | **string** comma-separated list | assigns a set of labels to a node [env=dev,app=web]. See the explanation of labeling mechanism in the [Labeling Nodes](admin-guide.mdx#labeling-nodes-and-applications) section.
```
Comment (webvictim) [inline docs/5.0/pages/cli-docs.mdx:44]:
```suggestion
| `--labels` | none | **string** comma-separated list | assigns a set of labels to a node, for example [env=dev,app=web]. See the explanation of labeling mechanism in the [Labeling Nodes](admin-guide.mdx#labeling-nodes-and-applications) section.
```

**Review by webvictim — APPROVED** [review]

File: docs/5.0/pages/cli-docs.mdx (hunk 1 above)
Comment (awly) [reply docs/5.0/pages/cli-docs.mdx:44]: also, remove the square brackets
users might think they are needed in the flag value

File: tool/teleport/common/teleport.go [hunk 2]
```diff
@@ -117,9 +117,9 @@ func Run(options Options) (executedCommand string, conf *service.Config) {
 	start.Flag("config-string",
 		"Base64 encoded configuration string").Hidden().Envar(defaults.ConfigEnvar).
 		StringVar(&ccf.ConfigString)
-	start.Flag("labels", "List of labels for this node").StringVar(&ccf.Labels)
+	start.Flag("labels", "Comma-separated list of labels for this node [env=dev,app=web]").StringVar(&ccf.Labels)
```
Comment (awly) [inline tool/teleport/common/teleport.go:120]:
```suggestion
	start.Flag("labels", "Comma-separated list of labels for this node, for example env=dev,app=web").StringVar(&ccf.Labels)
```

**Review by awly — APPROVED** [review]

**Review by klizhentas — CHANGES_REQUESTED** [review]
@stevenGravy please rebase the PR against `branch/v6`

**Pull request closed without merging**